app/metrics.yaml

```diff
@@ -4538,3 +4538,57 @@ engine_tab:
       - fenix-core@mozilla.com
       - skaspari@mozilla.com
     expires: "2021-12-31"
+
+android_keystore_experiment:
+  experiment:
+    type: event
+    description: |
+      General data about the functionality/experiment.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/17869
+    data_reviews:
+      - To be added
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-09-01"
+  get:
+    type: event
+    description: |
+      Tried to read from secure prefs.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/17869
+    data_reviews:
+      - To be added
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-09-01"
+  write:
+    type: event
+    description: |
+      Tried to write to secure prefs.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/17869
+    data_reviews:
+      - To be added
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-09-01"
+  reset:
+    type: event
+    description: |
+      The secure prefs file for the experiment was cleared.
```

Comment:
I'd reword this: "An experiment failed, and was reset to run again in the future from a blank state."